OpenNefia.Content/UI/Element/FeatWindow.cs

```diff
@@ -0,0 +1,311 @@
+                Graphics.SetBlendMode(BlendMode.Alpha);
+                GraphicsEx.SetColor(Love.Color.White);
+                AssetListBullet.Draw(UiText.X - XOffset - 5 + width - 20, UiText.Y - 0);
+            }
+
+
+            public override void Draw()
+            {
+                if (IndexInList % 2 == 0 && Data is not FeatData.FeatHeader)
+                {
+                    Love.Graphics.SetColor(UiColors.ListEntryAccent);
+                    Love.Graphics.Rectangle(Love.DrawMode.Fill, X - 1, Y, 650 - Offset, 18);
+                }
+                switch (Data)
+                {
+                    case FeatData.GainedFeat feat:
+                        GraphicsEx.SetColor(Core.Maths.Color.White);
+                        FeatIcons.DrawRegion($"{(int)feat.Prototype.FeatType}", X - 7, Y - 5);
+                        UiText.Draw();
+                        break;
+                    case FeatData.FeatHeader:
+                        UiText.Draw();
+                        break;
+                    case FeatData.Feat feat:
+                        GraphicsEx.SetColor(Core.Maths.Color.White);
+                        FeatIcons.DrawRegion($"{(int)feat.Prototype.FeatType}", X - 27, Y - 5);
+                        base.Draw();
+                        DescriptionText.Draw();
+                        break;
+                }
+            }
+        }
+
+        protected readonly IPrototypeManager _prototypeManager = default!;
+        private const int ItemsPerPage = 15;
+        private const int WindowHeight = 430;
+        private const int WindowWidth = 740;
+        public UiPagedList<FeatData> List;
+        private UiTextTopic NameTopic;
+        private UiTextTopic DetailTopic;
+        private UiText FeatCountText;
+
+        protected IAssetDrawable AssetInventoryIcons;
+        private IAssetDrawable AssetDecoFeatA;
+        private IAssetDrawable AssetDecoFeatB;
+        private IAssetDrawable AssetDecoFeatC;
+        private IAssetDrawable AssetDecoFeatD;
+
+        private Func<Dictionary<FeatPrototype, int>> GetFeatsFunc;
+        private Action<FeatData.Feat> SelectFeatAction;
+        private Func<int> GetAvailableCountFunc;
+
+        public FeatWindow(Func<Dictionary<FeatPrototype, int>> getFeatsFunc, Action<FeatData.Feat> selectFeatAction, Func<int> getAvailableCountFunc) 
+            : base(keyHintXOffset: 64)
+        {
+            _prototypeManager = IoCManager.Resolve<IPrototypeManager>();
+            AssetInventoryIcons = InventoryHelpers.MakeIcon(InventoryIcon.Feat);
+            AssetDecoFeatA = new AssetDrawable(Protos.Asset.DecoFeatA);
+            AssetDecoFeatB = new AssetDrawable(Protos.Asset.DecoFeatB);
+            AssetDecoFeatC = new AssetDrawable(Protos.Asset.DecoFeatC);
+            AssetDecoFeatD = new AssetDrawable(Protos.Asset.DecoFeatD);
+
+            NameTopic = new UiTextTopic(Loc.GetString("Elona.FeatMenu.NameTopic"));
```

Comment:
For the time being, do `[Localize("Topic.Name")]` instead.

Reply:
changed in b1407d8